.circleci/config.yml

```diff
@@ -72,6 +72,7 @@ jobs:
       - docker_login
       - run: echo "Releasing version" && ./gradlew printVersion
       - run: ./gradlew publish
+      - run: ./gradlew uploadArchives
```

Comment:
I thought we had migrated everything off circleci? I guess this repo was missed - would it make sense to migrate it before adding more circleci workflows?